content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md

```diff
@@ -0,0 +1,116 @@
+# download_shard
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/DownloadShard.cpp "Source")
+
+Instructs the server to download a specific [shard of historical ledger data](history-sharding.html) from an external source. Your `rippled` server must be [configured to store history shards](configure-history-sharding.html). [New in: rippled 1.1.0][]
+
+The external source must provide the shard as an [lz4-compressed](https://lz4.github.io/lz4/) [tar archive](https://en.wikipedia.org/wiki/Tar_(computing)) served via HTTPS. The contents of the archive must match the database type used for your shard store (NuDB or RocksDB).
+
+_The `download_shard` method is an [admin method](admin-rippled-methods.html) that cannot be run by unprivileged users._
+
+
+### Request Format
+
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "command": "download_shard",
+  "shards": [
+    {"index": 1, "url": "https://example.com/1.tar.lz4"},
+    {"index": 5, "url": "https://example.com/5.tar.lz4"}
+  ]
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+  "method": "download_shard",
+  "params": [
+    {
+      "shards": [
+        {"index": 1, "url": "https://example.com/1.tar.lz4"},
+        {"index": 5, "url": "https://example.com/5.tar.lz4"}
+      ]
+    }
+  ]
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+
+The request includes the following fields:
+
+| `Field`    | Type    | Description                                           |
+|:-----------|:--------|:------------------------------------------------------|
+| `shards`   | Array   | List of Shard Descriptor objects (see below) describing shards to download and where to download them. |
+| `validate` | Boolean | _(Optional)_ If `false`, skip validating the downloaded data. The default is `true`, which checks that the shard in the archive contains all the data object for the shard and the shard is part of the ledger history of the current validated ledger. |
+
+Each **Shard Descriptor object** in the `shards` array has the following fields:
+
+| `Field` | Type   | Description                                               |
+|:--------|:-------|:----------------------------------------------------------|
+| `index` | Number | The index of the shard to retrieve. In the production XRP Ledger, the oldest shard has index 1 and contains ledgers 32750-49133. The next shard has index 2 and contains ledgers 49133-65517, and so on. |
+| `url`   | String | The URL where this shard can be downloaded. The URL must start with `https://` and must end with `.tar.lz4` (not case-sensitive). |
+
+### Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "result": {
+    "message": "downloading shards 450",
```

Comment:
What does the "450" designate?

Reply:
The message is constructed as `downloading shards` + the shard indexes. In that particular example, 450 was the only shard index specified. The example could be better.

Reply:
oh, that's a response to a request to download shard number 450. The example request doesn't match the example response. (I should fix that.) But yeah, it looks like it says "shards" even if there's only one.